docs/commands.md

```diff
@@ -0,0 +1,96 @@
+<!--[metadata]>
++++
+title = "Understand the typical commands"
+description = "Description of key notary commands"
+keywords = ["docker, notary, notary-client"]
++++
+<![end-metadata]-->
+
+# Understand the typical notary commands
+
+## Terminology
+
+1. GUN: Notary uses Globally Unique Names (GUNs) to identify trust collections
```

Comment:
I was thinking we could add explanations of what the canonical roles are, what a target is, what a delegation is here? Thoughts on those + any additions? @cyli @endophage @riyazdf @NathanMcCauley